### ISSUE TITLE
tyrus-container-grizzly-client 1.15 depends on grizzly-framework 2.4.4 which is not available in maven central

### ISSUE THREAD

**Issue by cthiebaud** [eclipse-ee4j/tyrus#664]
tyrus-container-grizzly-client 1.15 depends on grizzly-framework 2.4.4 which is not available in maven central

in fat, it looks that the build e.g. https://ci.eclipse.org/tyrus/job/tyrus-master-release/72 copies these grizzly dependencies from some staging repo. Isn't that a bug ?

If it i not a bug., please document how to get the public release of grizzly 2.4.4

**Comment by cthiebaud** [comment]
Ok, grizzly-framework 2.4.4 is now public on maven central*, but if nothing is done the same issue is likely to occur at the next release(s), am I correct ?

[*] https://repo.maven.apache.org/maven2/org/glassfish/grizzly/grizzly-framework/2.4.4/

**Comment by bravehorsie** [comment]
All EE4J component releases went through the staging repository during integration testing. I suppose releasing to central in wrong order was an issue in this case instead of job script content. 

@Tomas-Kraus Can you confirm please?

**Comment by Tomas-Kraus** [comment]
Yes, grizzly-framework 2.4.4 should be released to central before anything that depends on it. Unfortunately whole Jakarta EE dependency graph is quite complicated so such issues may occur from time to time.
